clang/lib/DPCT/CallExprRewriterGraphicsInterop.cpp

```diff
@@ -0,0 +1,24 @@
+//===--------------- CallExprRewriterDriver.cpp ---------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "CallExprRewriter.h"
+#include "CallExprRewriterCommon.h"
+
+namespace clang {
+namespace dpct {
+
+void CallExprRewriterFactoryBase::initRewriterMapGraphicsInterop() {
+  RewriterMap->merge(
+      std::unordered_map<std::string,
+                         std::shared_ptr<CallExprRewriterFactoryBase>>({
+#include "APINamesGraphicsInterop.inc"
```

Comment:
Can we inline clang/lib/DPCT/APINamesGraphicsInterop.inc here?

Reply:
Can you please explain your reason for suggesting the inclusion the contents of the .inc file here directly?
The other CallExpr files seem to include the .inc header instead.